CHANGELOG.md

```diff
@@ -78,6 +78,10 @@
 [Earl Gaspard](https://github.com/earlgaspard)
 [#135](https://github.com/BottleRocketStudios/iOS-Hyperspace/pull/135)
 
+* Add `defaultMaxRecoveryAttempts` to `RequestDefaults` and use for `maxRecoveryAttempts` in `Request`.
```

Comment:
Might be good to also note that we changed the default from "unlimited retries" to "1 retry".

Reply:
Ok